### DISCUSSION TITLE
why my game crashs very ramdonly

### DISCUSSION THREAD

**Discussion by leonpano2006** [piotrulos/MSCModLoader#185] [Help me] [answered]
i just play then crashs 
reproduce steps is only play then crash(but right away but i didn't do anything spical)

before it does not happen but now yes(i didn't add any mods in meantime)
[output_log.txt](https://github.com/piotrulos/MSCModLoader/files/13653002/output_log.txt)

edit: this time crashed without doing anything
[output_log.txt](https://github.com/piotrulos/MSCModLoader/files/13653036/output_log.txt)

**Comment by piotrulos** [comment, marked as answer]
Looks like memory leak or something since last message is "Using memoryadresses from more that 16GB of memory" probably caused by mod or something

**Reply by leonpano2006** [reply]
But how to find what mod is that?
And why it crash by used more than 16GB RAM?
I have 256GB ram

**Reply by piotrulos** [reply]
It's old unity bug, that linux binary only works to 16GB.

**Reply by leonpano2006** [reply]
So fix is only use proton or use windows?

**Reply by leonpano2006** [reply]
> It's old unity bug, that linux binary only works to 16GB.

Btw
Is there anyway to find out what mod is leaking memory?

**Reply by leonpano2006** [reply]
And if I want to ask how did you port My summer car to Linux
Should I open new thread for this new question?

How to compile that thing?

And can I port other unity games in same way?

**Reply by piotrulos** [reply]
It's not really a port it's just that precompiled binaries worked with msc (it still works partially, since some shaders are broken)

**Reply by leonpano2006** [reply]
> It's not really a port it's just that precompiled binaries worked with msc (it still works partially, since some shaders are broken)

How did precompiled binary works on Linux?

Can I do this on other unity game?
Ex. Subnautica/bz phasmophobia...

**Reply by piotrulos** [reply]
It will not work on unity games compiled in c++, because then you need to have actual source code.
